d2l-table-shared-styles.js

```diff
@@ -18,36 +18,6 @@ $_documentContainer.innerHTML = `<custom-style>
 			--d2l-table-row-border-color-selected: var(--d2l-color-celestine);
 			--d2l-table-row-background-color-selected: var(--d2l-color-celestine-plus-2);
 
```

Comment:
I think we just went a little crazy with exposing these style mixins... not used.